### PR TITLE
New extension: geminiExtension.js

### PR DESCRIPTION
**Pull request by MicroboyLLC**
This extension allows using Google Gemini inside of Scratch projects (fun fact: the extension itself is generated by Google Gemini!). This can be useful in a lot of cases, like working chatbots, characters with realistic AI-powered dialogue that adapts to the things that happen in the game, and more.

### CODE REVIEW

File: extensions/-Microboy-/geminiExtension.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
+// Name: Gemini Extension
+// ID: geminiExtension
+// Description: Allows using Google Gemini within Scratch projects. In fact, this extension is generated by Google Gemini itself!
```
Comment (LilyMakesThings) [inline extensions/-Microboy-/geminiExtension.js:3]: > In fact, this extension is generated by Google Gemini itself!
We **seriously** prohibit extensions that are generated by AI and not created by someone who has any prior JavaScript experience. 

Language models do not understand how to make Scratch extensions, let alone TurboWarp extensions which have a whole other set of needs.

What you've done here is just admitted that you haven't read any of the TurboWarp extension documentation, nor the contributing guidelines. Quite a self report, don't you think?

For that very reason I'm not even going to review this.

**Comment by snowboyz0825** [comment]
Using your own personal API key is also dangerous and probably against some term of service somewhere

**Comment by MicroboyLLC** [comment]
I guess I'll make my own extension hosting service for my extensions.

**Comment by MicroboyLLC** [comment]
Also, I've read the guidelines and gave Google Gemini several different
examples and debug prompts to get it working, and they don't say anything
about AI-generated extensions.

**Pull request closed without merging**